README.md

```diff
@@ -0,0 +1,38 @@
+[![PyPi package link](https://img.shields.io/pypi/v/heavyai?style=for-the-badge)](https://pypi.org/project/heavyai/)
+[![Conda package link](https://img.shields.io/conda/vn/conda-forge/heavyai?style=for-the-badge)](https://anaconda.org/conda-forge/heavyai)
+
+
+heavyai
+=======
+
+This package enables using common Python data science toolkits with
+[HeavyDB](http://heavy.ai).
+It brings data frame support on CPU and GPU as well as support for arrow.
+See the [documentation](http://heavyai.readthedocs.io/en/latest/?badge=latest)
+for more.
+
+Quick Install (CPU)
+-------------------
+
+Packages are available on conda-forge and PyPI::
+
+   conda install -c conda-forge heavyai
+
+   pip install heavyai
+
+Quick Install (GPU)
+-------------------
+
+We recommend creating a fresh conda 3.8 or 3.9 environment when installing
+heavyai with GPU capabilities.
+
+To install heavyai for GPU Dataframe support (conda-only):
+
+```bash
+conda create -n heavyai-gpu -c rapidsai -c nvidia -c conda-forge -c defaults python=3.9 cudf cudatoolkit=11.0 heavyai
```

Comment:
This command gives me conda package conflicts but the following works:
```
conda create -n heavyai-gpu -c rapidsai -c nvidia -c conda-forge -c defaults python cudf cudatoolkit heavyai
```
FYI, this suggests installing python 3.8, cudatoolkit 11.4, arrow-cpp 5.0 but no pyheavydb! (because it is not available via conda yet).

Could we provide a temporary solution using `pip install pyheavydb` so that users could get a functional environment?

Reply:
Ok yes I will update. Strange that Python 3.9 is not working though. In the CI with the docker image from rapids we have 3.9.

Agreed for the temporary solution for conda with adding pip info.